fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/navbar/BottomToolbarContainerView.kt

```diff
@@ -55,6 +53,10 @@ class BottomToolbarContainerView(
                     }
                 }.value
 
+                // In future, this will also accept tab.content.canGoForward
+                // to manage active/inactive state of navigation buttons
+                val buttons = ActionItemListBuilder.build(location)
```

Comment:
if we're passing in the complex `NavBarLocation` to only make the `buttons` list, what if we instead pass `buttons` into `BottomToolbarContainerView` instead? That way, the View doesn't even need to know about some of the complex dependencies and their transformations.

Reply:
that's a good point, `BottomToolbarContainerView` could be _dummier_ and just receive buttons to display!